src/components/learning-guide-chart.coffee

```diff
@@ -97,25 +100,25 @@ module.exports = class LearningGuideChart
       .text("Show All #{guide.title}")
       .on('click', =>
         @displayTopic()
+        detailPane = document.querySelector('.learning-guide-chart .footer')
```

Comment:
not getting invariant error, see what you mean though.  I have some reservations as well; specifically some dom manipulations and ui changes need to wait until the elements are done being painted to get their attributes and using just using ReactDOM isnt enough.  If we can't do a simple css manipulation on an 'outer leaf' then we may have still have to select the dom node after painting the 'actual dom', for attributes relative to the document such as scrollHeight.